src/main/java/com/launchdarkly/openfeature/serverprovider/Provider.java

```diff
@@ -86,41 +89,120 @@ public Metadata getMetadata() {
     @Override
     public ProviderEvaluation<Boolean> getBooleanEvaluation(String key, Boolean defaultValue, EvaluationContext ctx) {
         EvaluationDetail<Boolean> detail
-                = this.client.boolVariationDetail(key, evaluationContextConverter.toLdContext(ctx), defaultValue);
+            = this.client.boolVariationDetail(key, evaluationContextConverter.toLdContext(ctx), defaultValue);
```

Comment:
I feel like formatting always changes even when I am using the same rules and the same tools, but at different points in time.

Reply:
IDEs are weird.